pkg/security/secl/compiler/eval/operators.go

```diff
@@ -352,7 +352,7 @@ func StringEquals(a *StringEvaluator, b *StringEvaluator, state *State) (*BoolEv
 
 		return &BoolEvaluator{
 			Value:           op(ea, eb),
-			Weight:          a.Weight + InArrayWeight*len(eb),
+			Weight:          a.Weight,
```

Comment:
Why is this only related to a and not b ?

Reply:
Removed a & b as they don't depend on a handler